core/src/main/resources/application.properties

```diff
@@ -234,7 +234,8 @@ klaw.shedlock.lockAtMostFor=PT60M
 klaw.shedlock.tablename=kwshedlock
 
 # log file settings
-logging.level.root=info
+logging.level.root=error
```

Comment:
do we want this always ?

Reply:
Ah damn this I need to revert, thanks for catching that. It was setup specifically for helping one user.